src/main/java/ch/njol/skript/structures/StructOptions.java

```diff
@@ -113,44 +113,33 @@ public String toString(@Nullable Event e, boolean debug) {
 		return "options";
 	}
 
-	/**
-	 * A method to obtain all options registered within a Script.
-	 * @param script The Script to obtain options from.
-	 * @return The options of this Script, or null if there are none.
-	 */
-	@Nullable
-	public static HashMap<String, String> getOptions(Script script) {
-		OptionsData optionsData = script.getData(OptionsData.class);
-		return optionsData != null ? optionsData.options : null;
-	}
-
-	/**
-	 * Replaces all options in the provided String using the options of the provided Script.
-	 * @param script The Script to obtain options from.
-	 * @param string The String to replace options in.
-	 * @return A String with all options replaced, or the original String if the provided Script has no options.
-	 */
-	public static String replaceOptions(Script script, String string) {
-		Map<String, String> options = getOptions(script);
-		if (options == null)
-			return string;
-
-		String replaced = StringUtils.replaceAll(string, "\\{@(.+?)\\}", m -> {
-			String option = options.get(m.group(1));
-			if (option == null) {
-				Skript.error("undefined option " + m.group());
-				return m.group();
-			}
-			return Matcher.quoteReplacement(option);
-		});
-
-		assert replaced != null;
-		return replaced;
-	}
-
-	private static final class OptionsData implements ScriptData {
+	public static final class OptionsData implements ScriptData {
+
+		private final HashMap<String, String> options = new HashMap<>(15);
```

Comment:
```suggestion
		private final Map<String, String> options = new HashMap<>(15);
```

Reply:
Nit-pick but entering 15 as parameter is redundant as the default is 16.